### DISCUSSION TITLE
Switch between themes

### DISCUSSION THREAD

**Discussion by gokhancetiner** [uicrooks/shopify-theme-lab#36] [Q&A]
Hello,

That is the case, there is more than one template on the store or let's say I want to add a new feature on the existing store. How the way should be? 

1) How to create a new template on an existing code? 
2) I duplicate the existing template on the store then I use the new template's id the id on the config .yml is it okay?

I just want to learn what is the best way.

Cheers,

**Comment by sergejcodes** [comment]
This project is not intended to work with themes developed by Shopify. I already answered, how you might be able to get it to run [here](https://github.com/uicrooks/shopify-theme-lab/discussions/31). You have to download the Theme from the store and set up a new project locally.

**Reply by gokhancetiner** [reply]
Hello @sergejcodes,

If this is the only way to do it's okay.

Cheers
